### PR TITLE
feat(graphql-server): implement graphql mesure stats resolver

### PR DESCRIPTION
**Pull request by tglatt**

### CODE REVIEW

**Comment by douglasduteil** [comment]
@tglatt rebase plz :)

**Comment by douglasduteil** [comment]
Might want to split the PR too :)

**Comment by kwiss** [comment]
> Might want to split the PR too :)

we can split client on a separate PR, @tglatt do you see something else to split

**Comment by douglasduteil** [comment]
knex seed changes too

**Comment by tglatt** [comment]
splitting in progress....

File: packages/graphql-server/src/resolvers/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { merge } from 'lodash';
-import hello from "./hello";
+import { merge } from "lodash";
+import mesureStatResolver from "./mesure-stat.resolver";
```
Comment (kwiss) [inline packages/graphql-server/src/resolvers/index.ts]: do we need to add .revolver ?

File: packages/graphql-server/src/resolvers/mesure-stat.resolver.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  closedMesureNumber,
+  mesureStateCategoryStatistics,
+  mesureTypeCategoryEvolution,
+  mesureTypeCategoryStatistics,
+  newMesureNumber
+} from "./query/mesure-stat";
+
+export const resolvers = {
+  Query: {
+    availableMesureNumber: async (
```
Comment (kwiss) [inline packages/graphql-server/src/resolvers/mesure-stat.resolver.ts]: we can remove that => https://github.com/SocialGouv/emjpm/issues/540 has been closed

File: packages/graphql-server/src/types/resolvers-types.ts [hunk 3]
```diff
@@ -20,140 +15,97 @@ export enum CacheControlScope {
...
-export type IDirectiveResolvers<ContextType = any> = DirectiveResolvers<
-  ContextType
->;
+export type QueryMesureTypeCategoryEvolutionArgs = {
+  start: Scalars["String"];
+  end: Scalars["String"];
+  region?: Maybe<Scalars["Int"]>;
+  department?: Maybe<Scalars["Int"]>;
+  court?: Maybe<Scalars["Int"]>;
+};
```
Comment (douglasduteil) [inline packages/graphql-server/src/types/resolvers-types.ts]: Is it a generated file ?

File: packages/graphql-server/src/resolvers/mesure-stat.resolver.ts (hunk 2 above)
Comment (tglatt) [reply packages/graphql-server/src/resolvers/mesure-stat.resolver.ts]: I do that in the next PR

File: packages/graphql-server/src/types/resolvers-types.ts (hunk 3 above)
Comment (tglatt) [reply packages/graphql-server/src/types/resolvers-types.ts]: https://graphql-code-generator.com/

**Pull request merged**